### PR TITLE
scripts: updated spelling.txt file

### PR DESCRIPTION
**Pull request by nashif**
We had wrong reports due to alteration of the file with correct words being
marked as misspelled.

Signed-off-by: Anas Nashif <anas.nashif@intel.com>

### CODE REVIEW

File: scripts/spelling.txt [hunk 1]
```diff
@@ -32,19 +33,22 @@ accoring||according
...
+accross||across
 acessable||accessible
 acess||access
 achitecture||architecture
 acient||ancient
 acitions||actions
 acitve||active
-acknowldegement||acknowldegement
-acknowledgement||acknowledgment
+acknowldegement||acknowledgment
```
Comment (cvinayak) [inline scripts/spelling.txt]: Perhaps this: acknowldegement||acknowledgement

**Review by cvinayak — CHANGES_REQUESTED** [review]

File: scripts/spelling.txt [hunk 2]
```diff
@@ -534,19 +608,28 @@ interruptted||interrupted
 interupted||interrupted
 interupt||interrupt
 intial||initial
+intialisation||initialisation
```
Comment (dbkinder) [inline scripts/spelling.txt]: previous spelling fixes (line 581) used the American spelling: initialization

File: scripts/spelling.txt [hunk 3]
```diff
@@ -600,16 +684,20 @@ messsage||message
 messsages||messages
 microprocesspr||microprocessor
 milliseonds||milliseconds
```
Comment (dbkinder) [inline scripts/spelling.txt]: I've also seen use of "milli seconds" (with a space) that should be "milliseconds"  (micro too)

**Comment by nashif** [comment]
I just copied this file as is from the Linux kernel, so i want to keep it as is, this british vs US is the least of my worries tbh :)

Maybe we should use canadian.

**Pull request merged**